doc/source/Getting_started/Troubleshooting.rst

```diff
@@ -159,26 +168,31 @@ Run your PyAEDT script. If the errors still persists, try the following:
     export ANSOFT_DEBUG_MODE=3
 
 
-Turn on the gRPC trace on the server side too:
+Enable the gRPC trace on the server:
 
 .. code:: console
 
     export GRPC_VERBOSITY=DEBUG
     export GRPC_TRACE=all
 
-Then start ansysedt.exe as gRPC server.
+Then start ansysedt.exe as a gRPC server and redirect the output.
```

Comment:
```suggestion
Then start the ``ansysedt.exe`` file as a gRPC server and redirect the output.
```

Reply:
```suggestion
Then run ansysedt.exe as a gRPC server and redirect the output.
```

Reply:
I understand the point, but a file is not started. The *.exe file is run. I added "the".